### DISCUSSION TITLE
Be smart, and chill out

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#190] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
If your some stupid 12 year old with no computer education don't go unenrolling and downgrading your chromebook. Just wait for an update, it will take like a month. Just chill out.

**Comment by 3kh0** [comment]
That's what I'm saying!

**Comment by The-Man-Is-Me** [comment]
If you are a kid with computer knowledge, do whatever, just don't get caught or blow up a chromebook.

**Reply by ghost** [reply]
still illegal though

**Reply by The-Man-Is-Me** [reply]
it's not illegal if you don't get caught

**Reply by The-Man-Is-Me** [reply]
JK, don't do illegal stuff

**Reply by Sxri** [reply]
> it's not illegal if you don't get caught

It's still considered illegal. I don't recommend doing any of these either, but if you're willing to risk the consequences and try to get away with it, thats on you.

**Reply by The-Man-Is-Me** [reply]
yep
